oops/hosts/galileo/ssi/__init__.py

```diff
@@ -214,14 +214,18 @@ def __init__(self, meta_dict):
         self.mode = meta_dict['TELEMETRY_FORMAT_ID']
 
         # Window
+        self.window = None
         if 'CUT_OUT_WINDOW' in meta_dict:
-            self.window = np.array(meta_dict['CUT_OUT_WINDOW'])
-            self.window_origin = self.window[0:2]-1
-            self.window_shape = self.window[2:]
-            self.window_uv_origin = np.flip(self.window_origin)
-            self.window_uv_shape = np.flip(self.window_shape)
-        else:
-            self.window = None
+            window = np.array(meta_dict['CUT_OUT_WINDOW'])
+            
+            # check for [-1,-1,-1,-1].  This is the value written in the 
+            # supplemental index when there is no CUT_OUT_WINDOW in the label.
+            if np.count_nonzero(window+1) != 0:         
```

Comment:
I find this line a bit obscure. How about `if np.all(window == -1)` instead? That's much more explicit about what you are testing.

Reply:
changed to `if np.all(window != -1)`